proto/regen/ecocredit/v1alpha1/tx.proto

```diff
@@ -172,6 +177,31 @@ message MsgRetireRequest {
 // MsgRetireRequest is the Msg/Retire response type.
 message MsgRetireResponse {}
 
+// MsgCancelRequest is the Msg/Cancel request type.
+message MsgCancelRequest {
+
+  // holder is the credit holder address.
+  string holder = 1;
+
+  // credits are the credits being cancelled.
+  repeated CancelUnits credits = 2;
+
+  // CancelUnits are the units of the batch being cancelled.
+  message CancelUnits {
+
+    // batch_denom is the unique ID of the credit batch.
+    string batch_denom = 1 [ (gogoproto.moretags) = "yaml:\"batch_denom\"" ];
+
+    // units are the units of credits being cancelled.
+    // Decimal values are acceptable within the precision returned by
+    // Query/Precision.
+    string units = 2;
```

Comment:
`amount`? we would also need to change other fields named `units` if we do this renaming

Reply:
Cool, I guess that's a good change to make now, as we've not launch the module yet